processing/data_collection/gazette/spiders/rs_gravatai.py

```diff
@@ -0,0 +1,63 @@
+from dateparser import parse
+from datetime import datetime
+
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RsGravataiSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4309209"
+    name = "rs_gravatai"
+    allowed_domains = ["gravatai.atende.net"]
+    start_urls = ["https://gravatai.atende.net/?pg=diariooficial"]
+
+    extra_editions_options = ("Suplementar", "Retificação")
+
+    def parse(self, response):
+        """
+        @url https://gravatai.atende.net/?pg=diariooficial
+        @returns requests 68
```

Comment:
The number of requests can change. Actually, right now there are 85 pages in the web site. I think you can update the documentation saying it will return a generator of the requests